sources/Context.cpp

```diff
@@ -202,11 +271,53 @@ Context::execute() {
   // Since DFL traces are persistent, they can be re-used after simulation is performed outside this function
   LOG(info) << MESS(SimulateInfo, basename_) << LOG_ENDL;
```

Comment:
This log and its comments are now relevant only for normal simulation: move them in the STEADY case